pkg/cmd/server/api/types.go

```diff
@@ -279,6 +279,21 @@ type MasterConfig struct {
 
 	// NetworkConfig to be passed to the compiled in network plugin
 	NetworkConfig MasterNetworkConfig
+
+	// JenkinsPipelineConfig holds information about the default Jenkins template
+	// used for JenkinsPipeline build strategy.
+	JenkinsPipelineConfig JenkinsPipelineConfig `json:"jenkinsPipelineConfig"`
```

Comment:
@jimmidyson will incorporate the fix with other changes I'm doing today, thanks for ping :)